packages/auth/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Auth",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Comment:
our house license is GPL-3 (as seen in the other package, you should just make this match them)